contrib/coreos/user-data.example

```diff
@@ -17,266 +17,13 @@ coreos:
-        docker exec deis-store-admin ceph osd out $OSD_ID
-        sleep 30
-        TIMEWAITED=0
-        until [[ $(docker exec deis-store-admin ceph health) == *"HEALTH_OK"* ]]
-        do
-          if [ $TIMEWAITED -gt "1200" ]
-          then
-            echo "ceph graceful removal timeout exceeded"
-            break
-          fi
-          echo "waiting" && sleep 5
-          TIMEWAITED=$((TIMEWAITED+5))
-        done
-        docker stop deis-store-daemon
-        docker exec deis-store-admin ceph osd crush remove osd.$OSD_ID
-        docker exec deis-store-admin ceph auth del osd.$OSD_ID
-        docker exec deis-store-admin ceph osd rm $OSD_ID
-        etcdctl rm /deis/store/osds/$PUBLIC_IP
-        etcdctl rm /deis/store/hosts/$PUBLIC_IP && sleep 10
-        # remove ceph mon
-        docker stop deis-store-monitor || true
-        docker exec deis-store-admin ceph mon remove `hostname -f` # fixme
-        docker stop deis-store-metadata || true
-      fi
-
-      # removing the node from etcd
-      NODE=$(etcdctl member list | grep `cat /etc/machine-id` | cut -d ':' -f 1)
-      etcdctl member remove $NODE
-  - path: /opt/bin/wupiao
-    permissions: '0755'
-    content: |
-      #!/usr/bin/env bash
-      # [w]ait [u]ntil [p]ort [i]s [a]ctually [o]pen
-      [ -n "$1" ] && \
-        until curl -o /dev/null -sIf http://${1}; do \
-          sleep 1 && echo .;
-        done;
-      exit $?
-  - path: /opt/bin/download-k8s-binary
-    permissions: '0755'
-    content: |
-      #!/usr/bin/env bash
-      export K8S_VERSION="v1.0.1"
-      mkdir -p /opt/bin
-      FILE=$1
-      if [ ! -f /opt/bin/$FILE ]; then
-        curl -sSL -o /opt/bin/$FILE https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/amd64/$FILE
-        chmod +x /opt/bin/$FILE
-      else
-        # we check the version of the binary
-        INSTALLED_VERSION=$(/opt/bin/$FILE --version)
-        MATCH=$(echo "${INSTALLED_VERSION}" | grep -c "${K8S_VERSION}")
-        if [ $MATCH -eq 0 ]; then
-          # the version is different
-          curl -sSL -o /opt/bin/$FILE https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/amd64/$FILE
-          chmod +x /opt/bin/$FILE
-        fi
-      fi
-  - path: /opt/bin/scheduler-policy.json
-    content: |
-      {
-          "kind": "Policy",
-          "apiVersion": "v1",
-          "predicates": [{"name": "PodFitsPorts"},{"name": "PodFitsResources"},{"name": "NoDiskConflict"},{"name": "MatchNodeSelector"},{"name": "HostName"}],
-          "priorities": [{"name": "LeastRequestedPriority","weight": 1},{"name": "BalancedResourceAllocation","weight": 1},{"name": "ServiceSpreadingPriority","weight": 2},{"name": "EqualPriority","weight": 1}]
-      }
+        ExecStart=/usr/bin/coreos-cloudinit --from-url=https://raw.githubusercontent.com/aespinosa/deis/move-static-cloud-config/contrib/coreos/cloud-config
```

Comment:
I just set this to my branch to pass the tests. This should be something canonical and managed synced with releases.